tests/nnstreamer_mux/runTest.sh

```diff
@@ -117,4 +117,36 @@ callCompareTest testsynch05_7.log testsynch05_7.golden 13-8 "Compare 13-8" 1 0
 callCompareTest testsynch05_8.log testsynch05_8.golden 13-9 "Compare 13-9" 1 0
 callCompareTest testsynch05_9.log testsynch05_9.golden 13-10 "Compare 13-10" 1 0
 
+gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} --gst-debug=tensor_merge:5 tensor_merge name=merge mode=linear option=2 silent=false sync_mode=basepad sync_option=0 ! multifilesink location=testsynch06_%1d.log multifilesrc location=\"testsequence03_%1d.png\" index=0 caps=\"image/png, framerate=(fraction)30/1\" ! pngdec ! tensor_converter ! merge.sink_0 multifilesrc location=\"testsequence03_%1d.png\" index=0 caps=\"image/png, framerate=(fraction)20/1\" ! pngdec ! tensor_converter ! merge.sink_1" 14 0 0 $PERFORMANCE
+callCompareTest testsynch06_0.log testsynch05_0.golden 14-1 "Compare 14-1" 1 0
+callCompareTest testsynch06_1.log testsynch05_1.golden 14-2 "Compare 14-2" 1 0
+callCompareTest testsynch06_2.log testsynch05_2.golden 14-3 "Compare 14-3" 1 0
+callCompareTest testsynch06_3.log testsynch05_3.golden 14-4 "Compare 14-4" 1 0
+callCompareTest testsynch06_4.log testsynch05_4.golden 14-5 "Compare 14-5" 1 0
+callCompareTest testsynch06_5.log testsynch05_5.golden 14-6 "Compare 14-6" 1 0
+callCompareTest testsynch06_6.log testsynch05_6.golden 14-7 "Compare 14-7" 1 0
+callCompareTest testsynch06_7.log testsynch05_7.golden 14-8 "Compare 14-8" 1 0
+callCompareTest testsynch06_8.log testsynch05_8.golden 14-9 "Compare 14-9" 1 0
+callCompareTest testsynch06_9.log testsynch05_9.golden 14-10 "Compare 14-10" 1 0
```

Comment:
Update argument order.
$1 should be the golden and $2 should be the log.
With the $5 == 1, the current code may say "PASS" when *.log is NULL.